README.md

```diff
@@ -137,7 +137,7 @@ function MyForm() {
 ### First time setup
 
 - Install node.js (only Node v20 is currently supported)
-- Install pnpm: [https://pnpm.io/installation](https://pnpm.io/installation) (use pnpm 8.6.8+)
```

Comment:
Maybe just change this to use semver instead of +? Idk if versions prior to 8.6.8 work

Reply:
Ditto what Drew said-- @NathanFarmer has been using successfully 9.4.0